pkg/apis/eventing/v1alpha1/trigger_types.go

```diff
@@ -80,7 +80,7 @@ type TriggerSpec struct {
 
 	// Subscriber is the addressable that receives events from the Broker that pass the Filter. It
 	// is required.
-	Subscriber *duckv1.Destination `json:"subscriber,omitempty"`
+	Subscriber duckv1.Destination `json:"subscriber"`
```

Comment:
Should the broker comment be updated to have +optional ?

Reply:
I don't think so, it's required, but it will be defaulted before getting persisted. In the CRD it's marked as optional. I think it's right the way it is.